app/controllers/admin/cms/collections_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Admin::Cms::CollectionsController < Admin::ApplicationController
+  def show
+  end
+
+  def new
+    @resource = Collection.new
+  end
+
+  def create
+    @resource = Collection.new(resource_params)
+
+    if @resource.save
+      flash[:notice] = t(".notice")
```

Comment:
how much do we care about I18n on this project? i see these two are in the `config/locales/en.yml` file, but i think every other bit of copy is hard-coded?